.github/workflows/scripts/create_npm_package_file.sh

```diff
@@ -15,6 +15,7 @@ cat <<EOF >> "package.json"
   "version": "$1",
   "description": "A zero dependency library for serializing data.",
   "main": "index.js",
+  "types": "types.d.ts",
```

Comment:
Because our `main` is `index.js` if we rename `types.d.ts` to `index.d.ts` then we don't even have to specify a types file:

> Also note that if your main declaration file is named index.d.ts and lives at the root of the package (next to index.js) you do not need to mark the "types" property, though it is advisable to do so.
https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html